games-action/cro_mag_rally/cro_mag_rally-3.0.1.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="Racing game set in prehistoric times"
+DESCRIPTION="In Cro-Mag Rally you are a speed-hungry caveman named Brog who races through \
+the Stone, Bronze, and Iron Ages in primitive vehicles such as the Geode Cruiser, Bone Buggy, \
+Logmobile, Trojan Horse, and many others. Brog has at his disposal an arsenal of primitive \
+weaponry ranging from Bone Bombs to Chinese Bottle Rockets and Heat Seeking Homing Pigeons. \
+In addition to single-player racing where one player races against the computer, there are \
+also several different multi-player modes including Tag, Capture the Flag, and Survival. \
+Up to four players can play on a single computer in split-screen mode"
+HOMEPAGE="https://github.com/jorio/CroMagRally"
+COPYRIGHT="2000 Pangea Software, Inc."
+LICENSE="CC-BY-NC-SA-4.0"
+REVISION="1"
+SOURCE_URI_1="https://github.com/jorio/CroMagRally/archive/refs/tags/3.0.1.zip"
```

Comment:
Use $portVersion here and in PATCHES:
```diff
-SOURCE_URI_1="https://github.com/jorio/CroMagRally/archive/refs/tags/3.0.1.zip"
+SOURCE_URI_1="https://github.com/jorio/CroMagRally/archive/refs/tags/$portVersion.zip"
```
Any reason why you used the zip archive instead of the tar.gz one (iirc tar.gz is smaller in size then zip)?

Reply:
No particular reason, IIRC I was closely following Bugdom recipe, as this game uses the same engine and many things are similar. I can change it to tar.gz

Reply:
You can leave it for now, maybe on a next change switch to tar.gz :) Only the $portVersion would be nice to include.

Reply:
Done